files/build_templates/sonic_debian_extension.j2

```diff
@@ -228,12 +228,14 @@ sudo cp -f $IMAGE_CONFIGS/bash/bash.bashrc $FILESYSTEM_ROOT/etc/
 # Install readline's initialization file
 sudo cp -f $IMAGE_CONFIGS/readline/inputrc $FILESYSTEM_ROOT/etc/
 
+sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y install libsystemd-dev -t bullseye-backports
```

Comment:
@Junchao-Mellanox do we need to have `libsystemd0` also?

Reply:
i will check this